### ISSUE TITLE
DON'T DO NOW - SEE COMMENT a11y: (Baking) "Table #.#" or "Figure #.#" should appear in the screen reader (add aria-label)

### ISSUE THREAD

**Issue by ambersdai** [openstax/cnx#122]
## Overview

COMMENT: We currently have a different implementation for this - we are moving the entire caption (including table/figure #) into the summary attribute. We would like to try this approach before doing the work to support the aria tag.

From https://drive.google.com/file/d/0BxiTbl6K_ZNRelpxRlkwYkZnb3c/view

Issues 6: Tables and figure structure is not accessible
Tables’ structure is not accessible: it is hard to identify them (1.3.1)

### Reference to standards
1.3.1 Info and Relationships: Information, structure, and relationships conveyed through presentation can be programmatically determined or are available in text. (Level A)
### Suggestion
Ensure that each table header in a data table refers to data cells so that each header cell that is used is actually a header of something. Make tables more accessible: give names to the tables (table 1, table 2) and put the main name of the table before the table not in the row (as is in PDF)
## PRs
What PRs need to be merged to complete this task/feature
## Test Plan
How will QA know this feature works? What parts of regression tests would be most affected?
## Acceptance Criteria
Expected behavior

![](https://trello-attachments.s3.amazonaws.com/581c83e5cae45416fa9bf5e2/5a7df66dc29a95a0c92a5f6c/8df11de20b8179c0ca3947f082c09cfd/image.png)

![](https://trello-attachments.s3.amazonaws.com/581c83e5cae45416fa9bf5e2/5a7df66dc29a95a0c92a5f6c/8a55b85b45622903cd163e91de4152bb/image.png)
